src/Stream/StreamerInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Findologic\Plentymarkets\Stream;
+
+use GuzzleHttp\Psr7\Response;
+
+interface StreamerInterface
+{
+    /**
+     * @param Response $response
+     * @return string
+     */
+    public function streamToFile(Response $response): string;
```

Comment:
This method is not necessary, it can be implemented in `streamToFileFromStreamedFile` since:

```
$stream = $response->getBody(); 
$streamInfo = $stream->getMetadata();
```
$streaminfo['uri'] has the filename associated to the stream.
Check: https://www.php.net/manual/en/function.stream-get-meta-data.php